ui/src/main/webapp/css/windup-web.css

```diff
@@ -161,13 +161,15 @@ fieldset.fields-section-pf {
 }
 
 .wu-horizontal-nav-content {
-    margin-top: 122px;
+/*    margin-top: 122px; */
```

Comment:
I think this is just in case we find out that there had been a reason to use the other. I haven't seen any issues so far, though.